app/controllers/explorer.js

```diff
@@ -121,7 +121,9 @@ export default class ExplorerController extends Controller {
   set source(newSource) {
     const { id } = newSource;
 
-    this.transitionToRoute('explorer', { queryParams: { source: id }});
+    sessionStorage.setItem('source', newSource.id)
```

Comment:
It might be more trouble than it's worth but I think we should consider alternatives to using sessionStorage for this. Keeping track of "state" like this is the sort of thing we should use Ember for.

Reply:
I'm open to alternatives but from my research I didn't see many state management options that wouldn't significantly add to the scope of the ticket (but I'm also not super familiar with Ember yet so there may well be)

Reply:
My thoughts on the options: 
1. Application controller - would not preserve state if you refresh the application
2. URLs - Preserves state across reloads and forward/backwards page navigation only if user strictly uses the same method to move between pages _both_ ways. Ie strictly uses application controls or strictly uses browser navigation. Requires you to add to URLs used for transitioning back to Map page. Cons: URL looks messy and awkward on the map page. 
3. Ember service - doesn't preserve state if you refresh the application
4. Local storage - preserves state if you refresh application. User can use a mix of browser and application navigation without worry of losing state. URL stays clean on the map page.

Reply:
@godfreyyeung thanks for laying those options out. Tbh I'm not sure if maintaining across application refreshes is something we want necessarily? Users might hit refresh specifically to "clear" state like this. I'm cool with eliminating a service as an option, I agree with the point you made earlier that the Ember docs seem suggest Services aren't intended to keep this kind of state.

Reply:
The issue is that in order for this to work we will need to preserve state across refreshes somehow, since clicking into any of the links in the nav bar does cause a refresh in the app